api/v1beta2/integrationtestscenario_webhook.go

```diff
@@ -53,6 +53,28 @@ func (r *IntegrationTestScenario) ValidateCreate() (warnings admission.Warnings,
 					"param manually defined because it will be automatically generated"+
 					"by the integration service")
 		}
+		// we won't enable ITS if git resolver with url & repo+org
+		urlResolverExist := false
+		repoResolverExist := false
+		orgResolverExist := false
+
+		for _, gitResolverParam := range r.Spec.ResolverRef.Params {
+			if gitResolverParam.Name == "URL" && gitResolverParam.Value != "" {
+				urlResolverExist = true
+			}
+			if gitResolverParam.Name == "REPO" && gitResolverParam.Value != "" {
+				repoResolverExist = true
+			}
+			if gitResolverParam.Name == "ORG" && gitResolverParam.Value != "" {
+				repoResolverExist = true
+			}
+			if urlResolverExist && repoResolverExist && orgResolverExist {
```

Comment:
This will only fail if all three parameters exist.  What if we have URL, Repo, and no org?  Do we continue silently and just use the URL?  Do we fail because repo was provided with no org?   Something else?

Reply:
That's a good point, I think that Tekton would fail even if just one of the repo and org was defined alongside the URL.
Maybe something like this would work better:
```
if urlResolverExist && (repoResolverExist || orgResolverExist) {
```

Reply:
yep , 
thanks @14rcole  for this important comment, 
agree with @dirgim suggestion, looks good enough